hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -1379,6 +1379,7 @@ int QuectelNcpClient::registerNet() {
             CHECK_TRUE(r == 1, SYSTEM_ERROR_UNKNOWN);
             r = CHECK_PARSER(respNwMode.readResult());
             CHECK_TRUE(r == AtResponse::OK, SYSTEM_ERROR_UNKNOWN);
+        #if PLATFORM_ID == PLATFORM_MSOM
```

Comment:
If we do want to enable 2G fallback on Tracker, we have enough space for 5.5.0. Just not in develop for now.